backend/dataall/modules/metadata_forms/services/metadata_form_service.py

```diff
@@ -0,0 +1,74 @@
+from dataall.base.context import get_context
+from dataall.base.db import exceptions, paginate
+from dataall.core.organizations.db.organization_repositories import OrganizationRepository
+from dataall.core.environment.db.environment_repositories import EnvironmentRepository
+
+from dataall.modules.metadata_forms.db.enums import MetadataFormVisibility
+from dataall.modules.metadata_forms.db.metadata_form_repository import MetadataFormRepository
+
+
+class MetadataFormParamValidationService:
+    @staticmethod
+    def validate_create_form_params(data):
+        visibility = data.get('visibility', MetadataFormVisibility.Team.value)
+        if not MetadataFormVisibility.has_value(visibility):
+            data['visibility'] = MetadataFormVisibility.Team.value
+
+        if 'SamlGroupName' not in data:
+            raise exceptions.RequiredParameter('SamlGroupName')
+
+        if 'homeEntity' not in data and (
+            visibility == MetadataFormVisibility.Organization.value
+            or visibility == MetadataFormVisibility.Environment.value
+        ):
```

Comment:
we should also ensure `homeEntity` has a value for `MetadataFormVisibility.Team.value` right?